### DISCUSSION TITLE
Not full instalation guide

### DISCUSSION THREAD

**Discussion by Gotovskiy** [chakra-ui/chakra-ui#9250] [General] [answered]
### Description

When installing at the third stage, I had problems with the fact that the ChakraProvider component requires a value - and there are no words about this in the documentation, manually look at the solutions on stackOverflow - please describe the installation in more detail.

### Link to Reproduction

https://stackblitz.com/edit/chakra-ui-v3-3xwnia?file=src%2Fmain.tsx

### Steps to reproduce

create vite app with react - install chakra-ui & add provider

### Chakra UI Version

3.2

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

**Comment by segunadebayo** [comment, marked as answer]
`ChakraProvider` needs a system in `value`. Pass `defaultSystem`, or the system you built with `createSystem`.

The Vite and Next guides show this now: https://chakra-ui.com/docs/get-started/frameworks/vite
